lib/config/httpRoutes.js

```diff
@@ -44,6 +44,7 @@ const routes = [
   { verb: 'get', path: '/:index/:collection/_search', controller: 'document', action: 'search' },
   { verb: 'get', path: '/:index/:collection/_specifications', controller: 'collection', action: 'getSpecifications' },
   { verb: 'get', path: '/validations/_scroll/:scrollId', controller: 'collection', action: 'scrollSpecifications' },
+  { verb: 'get', path: '/:index/_list&type=realtime', controller: 'collection', action: 'list', deprecated: { since: '2.1.4', message: 'Use this route instead: http://kuzzle:7512/_listSubscriptions' } }, // @deprecated
```

Comment:
Doesn't work if the invoked URL isn't exactly that one.

For instance, this URL won't return the deprecation notice because of the `pretty` argument:

`http://localhost:7512/foo/_list?pretty&type=realtime`

Also doesn't work if the API action is invoked using another protocol than HTTP.

The way I see the linked issue, the whole `type` argument should be deprecated, meaning that the default output (which uses `type=all`) should also print a meaningful message.

About the message: it should be more explicit: what is deprecated exactly?
Something along these lines could do: `The list of realtime collections has been deprecated since Kuzzle version <version>, and might be removed in a future major version. To get a list of realtime collections, use this API action instead: "realtime:list".`

Also, why is it deprecated since Kuzzle 2.1.4? :thinking: 
We should never deprecate a feature retroactively, it's confusing for users.

Lastly, the documentation should also be updated, explaining what is deprecated, why, since which version, and what to use instead.

Reply:
About why 2.1.4 and updating the doc, it's because it's already deprecated see [here](https://docs.kuzzle.io/core/2/api/controllers/collection/list/#optional), I'll correct the other points

Reply:
I think we have a design flaw here because it was only the pagination options that were deprecated with the `2.1.4`. The badges should be put on the corresponding options lines